### DISCUSSION TITLE
How to remove Vercel branding from web app (I cloned Git repo locally, want to customize)

### DISCUSSION THREAD

**Discussion by vdutts7** [vercel/community#1536] [Help] [answered]
### Summary

 I deployed a Vercel template OUTSIDE of Vercel -- cloned it locally using the [Github repo](https://github.com/Nutlope/twitterbio) that was linked. I modified it heavily (mainly for personal use / sharing with my family). Is there a way to customize the UI/UX further and remove all the clunky Vercel branding? There are headers I can't get rid of, background colors I can't change, and a lot of other issues like that. I noticed the Git repo README.md mentioned to specifically use a .env.local as opposed to the normal .env .... this is an unfamiliar area for me. Help would be appreciated. Best.

**Comment by amyegan** [comment, marked as answer]
Hi Vivek! That repo uses [Tailwind](https://tailwindcss.com/), so the styles can be changed by editing the HTML element classes in components and pages. You can change or remove the [Vercel logo in the header component](https://github.com/Nutlope/twitterbio/blob/2a55bb8fdc415ddbd6c70f5dfaf8c855c59e5e6e/components/Header.tsx#L24). And the [favicon](https://github.com/Nutlope/twitterbio/blob/2a55bb8fdc415ddbd6c70f5dfaf8c855c59e5e6e/public/favicon.ico) that appears in the browser tab can be replaced in the `/public` directory

**Comment by vdutts7** [comment]
Got it thanks! Additional question: how do I make sure / see if my
deployments on Vercel are publicly accessible and/ or visible? I connected
via Github and used a private repository.  I am worried about a mal-actor
who has threatened to try to sabotage my app and see my code and as my IP,
i really want to protect that and know what I can do. Please advise. Best.
It is regarding my account I operate on the email ***@***.***
***@***.***> * which I will CC here as well.

Best.

Vivek

On Wed, Feb 15, 2023 at 6:42 AM Amy ***@***.***> wrote:

> Hi Vivek! That repo uses Tailwind <https://tailwindcss.com/>, so the
> styles can be changed by editing the HTML element classes in components and
> pages. You can change or remove the Vercel logo in the header component
> <https://github.com/Nutlope/twitterbio/blob/2a55bb8fdc415ddbd6c70f5dfaf8c855c59e5e6e/components/Header.tsx#L24>.
> And the favicon
> <https://github.com/Nutlope/twitterbio/blob/2a55bb8fdc415ddbd6c70f5dfaf8c855c59e5e6e/public/favicon.ico>
> that appears in the browser tab can be replaced in the /public directory
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/1536#discussioncomment-4984195>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APIHEYK2MALZI4ZNDQRHOFDWXTTL7ANCNFSM6AAAAAAU4HEN4Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Vivek Dutta
UC Berkeley Class of 2023
Electrical Engineering & Computer Science (EECS) B.S.
***@***.***

**Reply by amyegan** [reply]
You can make use of basic [Deployment Protection](https://vercel.com/docs/concepts/deployments/deployment-protection#) features, which include [Vercel Authentication](https://vercel.com/docs/concepts/deployments/deployment-protection#vercel-authentication), the ability to [make preview URLs private](https://vercel.com/docs/concepts/deployments/preview-deployments#sharing-previews), and [shareable links](https://vercel.com/docs/concepts/deployments/deployment-protection#shareable-links). These features can help you keep your preview deployments private and avoid public access.

If you are on a Pro plan, you can also add a few [advanced security features](https://vercel.com/docs/concepts/deployments/advanced-deployment-protection#pricing) if you need [Private Production Deployments](https://vercel.com/docs/concepts/deployments/advanced-deployment-protection#private-production-deployments) and [Password Protection](https://vercel.com/docs/concepts/deployments/advanced-deployment-protection#password-protection)
